docs/development/getting_started.rst

```diff
@@ -0,0 +1,186 @@
+
+       $ npm install -g gulp
+       $ npm install
+       $ gulp sass
+
+5. Activate your virtual environment and create the statics and database:
+
+   .. code-block:: bash
+
+       $ source env/bin/activate
+       $ python src/manage.py collectstatic --link
+       $ python src/manage.py migrate
+
+6. Create a superuser to access the management interface:
+
+   .. code-block:: bash
+
+       $ python src/manage.py createsuperuser
+
+7. You can now run your installation and point your browser to the address
+   given by this command:
+
+   .. code-block:: bash
+
+       $ python src/manage.py runserver
+
+8. Create a .env file with database settings. See ``.env-example`` for an
+   example:
+
+   .. code-block:: bash
+
+       $ cp .env-example .env
+
+
+**Note:** If you are making local, machine specific, changes, add them to
+``src/openzaak/conf/includes/local.py``. You can base this file on the
+example file included in the same directory.
+
+**Note:** You can run watch-tasks to compile `Sass`_ to CSS and `ECMA`_ to JS
+using `gulp`_. By default this will compile the files if they change.
+
+.. _ECMA: https://ecma-international.org/
+.. _Sass: https://sass-lang.com/
+.. _gulp: https://gulpjs.com/
+
+
+Update installation
+-------------------
+
+When updating an existing installation:
+
+1. Activate the virtual environment:
+
+   .. code-block:: bash
+
+       $ cd open-zaak
+       $ source env/bin/activate
+
+2. Update the code and libraries:
+
+   .. code-block:: bash
+
+       $ git pull
+       $ pip install -r requirements/dev.txt
+       $ npm install
+       $ gulp sass
+
+3. Update the statics and database:
+
+   .. code-block:: bash
+
+       $ python src/manage.py collectstatic --link
+       $ python src/manage.py migrate
+
+
+Testsuite
+---------
+
+To run the test suite:
+
+.. code-block:: bash
+
+    $ python src/manage.py test openzaak
+
+Configuration via environment variables
+---------------------------------------
+
+A number of common settings/configurations can be modified by setting
+environment variables. You can persist these in your ``local.py`` settings
+file or as part of the ``(post)activate`` of your virtualenv.
+
+* ``SECRET_KEY``: the secret key to use. A default is set in ``dev.py``
+
+* ``DB_NAME``: name of the database for the project. Defaults to ``open-zaak``.
+* ``DB_USER``: username to connect to the database with. Defaults to ``open-zaak``.
+* ``DB_PASSWORD``: password to use to connect to the database. Defaults to ``open-zaak``.
+* ``DB_HOST``: database host. Defaults to ``localhost``
+* ``DB_PORT``: database port. Defaults to ``5432``.
+
+* ``SENTRY_DSN``: the DSN of the project in Sentry. If set, enabled Sentry SDK as
+  logger and will send errors/logging to Sentry. If unset, Sentry SDK will be
+  disabled.
+
+Settings
+========
+
+All settings for the project can be found in
+``src/openzaak/conf``.
+The file ``includes/local.py`` overwrites settings from the base configuration.
+
+Commands
```

Comment:
Boilerplate section that's wrong (there are custom commands) and doesn't actually say anything. I'd completely remove it tbh.